inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/InlongGroupServiceImpl.java

```diff
@@ -110,11 +109,9 @@ public String save(InlongGroupRequest groupInfo, String operator) {
         // Processing inlong group and extended information
```

Comment:
groupId need to get value from InlongGroupRequest.
Not fixed value "String groupId = "b_" + groupName.toLowerCase(Locale.ROOT);"

Reply:
I noticed this phenomenon, I will fix this in another feature.

Reply:
groupId get value from InlongGroupRequest at first.
If it is empty, then put the fixed vlaue to groupId.
It can be compatible with history.